sources/api/netdog/src/cli/mod.rs

```diff
@@ -47,6 +54,25 @@ where
     Ok(())
 }
 
+/// Search for the network configuration file and return the configuration if it parses successfully,
+/// otherwise return an error
+fn check_net_config() -> Result<Option<Box<dyn Interfaces>>> {
```

Comment:
As previously mentioned, I don't think we need or want to share this logic.  This logic is specific to a running Bottlerocket system because of the assumed paths.